main.html

```diff
@@ -65,27 +65,39 @@ <h3>Manage your HQ</h3>
 
   // Tank
   var tanks = PlayerStats.getAllTanksOwnedByPlayer()
-  var tankLooup = {}
+  var tankIds = []
+
   for (var tank of tanks)
   {
-    tankLooup[tank.index] = tank.shopImage
     var image = document.createElement("img");
     image.setAttribute("src", tank.shopImage);
+    image.setAttribute("id", tank.index);
+    tankIds.push(tank.index)
+    image.onclick = (event) => {
+      selectedTankIndex = tankIds[event.srcElement.id]
+      PlayerStats.setTankId(selectedTankIndex)
+    }
+    var nameText = document.createElement("label")
```

Comment:
Move this two rows below column.appendChild(image) to have creation and appending of label closer together